charts/cluster-autoscaler/values.yaml

```diff
@@ -95,6 +95,9 @@ azureClusterName: ""
 # Required if `cloudProvider=azure`
 azureNodeResourceGroup: ""
 
+# azureUseWorkloadIdentityExtension -- Whether to use Azure's workload identity extension for credentials.
+azureUseWorkloadIdentityExtension: false
+
 # azureUseManagedIdentityExtension -- Whether to use Azure's managed identity extension for credentials. If using MSI, ensure subscription ID, resource group, and azure AKS cluster name are set.
 azureUseManagedIdentityExtension: false
```

Comment:
Only one of these can be used at a time (if both are set, the current implementation will use workload identity). Please document (and consider adding a check in code that both are not set at the same time).

Reply:
I added an error in the code, and added documentation that you should not set both